### PR TITLE
feat: sdk reporting flag and e2e test

### PR DESCRIPTION
**Pull request by sjaanus**
1. Add flag
2. Add e2e test with more complete example
3. Some bug fixes

### CODE REVIEW

File: src/lib/features/project/project-store.ts [hunk 1]
```diff
@@ -742,15 +742,17 @@ class ProjectStore implements IProjectStore {
                     instances: [row.instance_id],
                     sdks: [getSdk(sdkParts)],
                 };
-                entriesMap.set(row.feature_name, entry);
+                entriesMap.set(row.app_name, entry);
```
Comment (FredrikOseberg) [inline src/lib/features/project/project-store.ts:745]: This change seems significant. I assume it's intended.

File: src/lib/features/project/project-store.ts [hunk 2]
```diff
@@ -742,15 +742,17 @@ class ProjectStore implements IProjectStore {
...
                 orderedEntries.push(entry);
             }
 
             const sdk = entry.sdks.find((sdk) => sdk.name === sdkParts[0]);
             if (!sdk) {
                 entry.sdks.push(getSdk(sdkParts));
             } else {
-                sdk.versions.push(sdkParts[1]);
+                if (!sdk.versions.includes(sdkParts[1])) {
```
Comment (FredrikOseberg) [inline src/lib/features/project/project-store.ts:753]: We never check if sdkParts exists after splitting, can there be a scenario where we don't have it?

**Review by FredrikOseberg — APPROVED** [review]
Approved, but some questions remain

File: src/lib/features/project/project-store.ts (hunk 1 above)
Comment (sjaanus) [reply src/lib/features/project/project-store.ts:745]: Yes, this was typo basically, which was caught by test.

File: src/lib/features/project/project-store.ts (hunk 2 above)
Comment (sjaanus) [reply src/lib/features/project/project-store.ts:753]: That is good question, also we might not have the value at all. Probably we need to add more validations in future.

**Pull request merged**